vip_spec.xsd

```diff
@@ -244,11 +249,11 @@ reference elements for polling location
         <xs:element name="Contact">
           <xs:complexType>
             <xs:sequence>
-              <xs:element name="Url" type="xs:anyURI" minOccurs="0" maxOccurs="unbounded" />
+              <xs:element name="Uri" type="xs:anyURI" minOccurs="0" maxOccurs="unbounded" />
               <xs:element name="AddressLine" type="xs:string" minOccurs="0" maxOccurs="unbounded" />
               <xs:element name="Email" type="xs:string" minOccurs="0" maxOccurs="unbounded" />
               <xs:element name="Fax" type="xs:string" minOccurs="0" maxOccurs="unbounded" />
-              <xs:element name="HoursOpenID" type="xs:IDREF" minOccurs="0" />
+              <xs:element name="Schedule" type="xs:IDREF" minOccurs="0" />
```

Comment:
It seems like this should be `HoursOpenID` to match our style guide?  See [here](https://github.com/votinginfoproject/vip-specification/blob/vip5/docs/style_guide.md#ids).